python/shotgun_fields/date_and_time_widget.py

```diff
@@ -68,3 +53,82 @@ def _create_human_readable_timestamp(self, dt, postfix=""):
 
         time_str = dt.strftime(format)
         return time_str
+
+    def _string_value(self, value):
+        """
+        Convert the Shotgun value for this field into a string
+
+        :param value: The value to convert into a string
+        :type value: :class:`datetime.datetime` or a float representing unix time
+        """
+
+        # shotgun_model converts datetimes to floats representing unix time so
+        # handle that as a valid value as well
+        if not isinstance(value, datetime.datetime):
+            value = datetime.datetime.fromtimestamp(value)
+        return self._create_human_readable_timestamp(value, " %I:%M%p")
```

Comment:
is it possible / does it make sense to have a tooltip that shows the exact value. I think this is how the activity stream works and i like how in github it says "you committed this _10 days ago_" and if you hover over the date you see the exact timestamp.